iModelJsNodeAddon/IModelJsNative.cpp

```diff
@@ -1788,6 +1788,33 @@ struct NativeDgnDb : BeObjectWrap<NativeDgnDb>, SQLiteOps
         return Napi::Number::New(Env(), (int)result);
         }
 
+    Napi::Value ConvertEC2XmlSchemas(NapiInfoCR info)
+        {
+        RequireDbIsOpen(info);
+        REQUIRE_ARGUMENT_STRING_ARRAY(0, ec2XmlStrings);
+        REQUIRE_ARGUMENT_ANY_OBJ(1, jsOpts);
+
+        JsInterop::SchemaImportOptions options;
+        options.m_schemaLockHeld = jsOpts.Get(JsInterop::json_schemaLockHeld()).ToBoolean();
+        const auto& maybeEcSchemaContextVal = jsOpts.Get(JsInterop::json_ecSchemaXmlContext());

```

Comment:
I don't see anybody calling this with a context, what's the scenario, try to cover it in a test

Reply:
We decided to exclude the ECSchemaReadContext from the parameter list since the caller no longer needs to provide it. Instead, the conversion logic creates its own schema context and uses it for the schema conversion process.